modules/gapi/test/own/mat_tests.cpp

```diff
@@ -384,4 +384,37 @@ TEST(OwnMat, ROIView)
     << to_ocv(roi_view) << std::endl
     << expected_cv_mat  << std::endl;
 }
+
+TEST(OwnMat, CreateOwnMat)
```

Comment:
Just figured out that the title of this test doesn't reflect the meaning of this test.

Also you can split this test into two or three, highlighting what aspects do you verify with this test.

Reply:
Fixed. Divided test into three. Changed names for tests.